src/coreComponents/physicsSolvers/fluidFlow/wells/CompositionalMultiphaseWellKernels.cpp

```diff
@@ -1760,10 +1790,23 @@ RateInitializationKernel::
       }
       else
       {
-        connRate[iwelem] = LvArray::math::min( 0.1 * targetTotalRate * totalDens[iwelem][0], 1e3 );
+        if ( isZero(targetMassRate) )
```

Comment:
same here for producer
(i understand that fixed mass rate is for injection but then we need to check it?)

Reply:
sorry just found that you do have that check
please ignore the comment above